tools/ScenarioTest.ResourceManager/Constants.cs

```diff
@@ -59,35 +59,6 @@ public class Category
 
         public const string OneSDK = "OneSDK";
 
-        // Below is the list of aliases to contact on test behavior
-
-        // Virtual Appliance dev team
-        public const string nvadev = "nvadev";
-
-        // SDN NRP Dev Team
-        public const string sdnnrp = "sdnnrp";
-
-        // Pankaj's Team
-        public const string pgtm = "pgtm";
-
-        // Windows Azure SLB Dev Team
-        public const string slbdev = "slbdev";
-
-        // Brooklyn FTEs
-        public const string brooklynft = "brooklynft";
-
-        // Azure Network Analytics Dev Team
-        public const string netanalyticsdev = "netanalyticsdev";
-
-        // Windows Azure NRP dev team
-        public const string wanrpdev = "wanrpdev";
-
-        //Azure NRP Firewall dev team
-        public const string azurefirewall = "azurefirewall";
```

Comment:
Hi @number213 are these variables also duplicated?

Reply:
OK, there is not any reference of these variables in the code. They can be remove safely.